tools/failures.py

```diff
@@ -68,65 +74,6 @@ def communicate(failures, to_insert, total_detected, testmode, date):
                    " changes from previous day", change, admin=True, results=True)
 
 
-def format_in_table(active_jobs, master):
```

Comment:
I remember we had remove it before, due to  we are storing the high value jobs now :)